README.md

```diff
@@ -23,9 +23,16 @@ The following table lists the configurable parameters of the Trino chart and the
 | `server.config.http.port` |  | `8080` |
 | `server.config.query.maxMemory` |  | `"4GB"` |
 | `server.config.query.maxMemoryPerNode` |  | `"1GB"` |
+| `server.config.query.maxTotalMemoryPerNode` |  | `"2GB"` |
+| `server.config.memory.heapHeadroomPerNode` |  | `"1GB"` |
 | `server.jvm.maxHeapSize` |  | `"8G"` |
```

Comment:
I think I prefer defaults, (maybe should be larger?), I mean when preparing deployment I would always look at default values.yaml file instead of readme, and I like to find all values there so I can easily copy/paste and overwrite whatever I need.
As a side note witch such defaults I can test chart using `ct install` without creating any more files :)

Reply:
In Helm the convention is that there should be a default for a variable if it is configurable. Maybe that's not the best design, but it's what users would expect. The goal that the creators had was for a user to be able to `helm install` as easy as you can `pip install`.

Reply:
Consider this resolved then. It makes more sense to do what other tools are doing (providing defaults).